org.opent2t.sample.lamp.superpopular/com.insteon.lightbulb/js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "opent2t-translator-com-insteon-lightbulb",
+    "version": "0.1.0",
+    "description": "Insteon Light Bulb",
+    "homepage": "http://opentranslatorstothings.org",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/opent2t/translators.git"
+    },
+    "main": "thingTranslator.js",
+    "license": "MIT",
+    "scripts": {
+        "test": "ava"
+    },
+    "ava": {
+        "files": [
+            "tests/*.js"
+        ]
+    },
+   "dependencies": {
+        "opent2t": "*",
+        "opent2t-translator-com-insteon-hub": "*"
+   },
+    "devDependencies": {
+        "optimist": "*",
+        "opent2t-translator-com-insteon-hub": "*",
+        "jshint": "^2.9.2",
```

Comment:
Removed.

---
In reply to: [82905961](https://github.com/openT2T/translators/pull/107#discussion_r82905961) [](ancestors = 82905961)